homeassistant/components/roborock/coordinator.py

```diff
@@ -59,6 +59,7 @@ def __init__(
 
         if mac := self.roborock_device_info.network_info.mac:
             self.device_info[ATTR_CONNECTIONS] = {(dr.CONNECTION_NETWORK_MAC, mac)}
+        self.maps: dict[int, str] = {}
```

Comment:
What exactly is this a dict of?

Reply:
Map flag to map name. Added as a comment